lib/roman_numerals.rb

```diff
@@ -1,19 +1,21 @@
 class ArabicToRoman
   def convert(number)
-    if number == 1
-      "I"
-    elsif number == 2
-      "II"
-    elsif number == 3
-      "III"
-    elsif number == 5
-      "V"
-    elsif number == 6
-      "VI"
-    elsif number == 7
-      "VII"
-    else        
-      "VIII"
+
+    romanValue = ""
+    remaining_number = number
+
+    if remaining_number >= 10
+    	romanValue << "X"
+    	remaining_number -= 10
     end
+
+	if remaining_number >= 5
+	  romanValue << "V"
+	  remaining_number -= 5
+	end
+	   
+    romanValue << "I" * remaining_number
```

Comment:
Perhaps at this point, you could've exposed the duplication?
Make it similar to the two above.

```ruby
if remaining_number >= 1
 	  romanValue << "I"
 	  remaining_number -= 1
end
```